test/minikube/minikube_long_restore_test.go

```diff
@@ -1010,8 +1000,6 @@ func TestCornerCaseKubernetesSnapshotRestore(t *testing.T) {
 
 	// Test restoring an archive with a journal embedded
 	t.Run("testArchive", func(t *testing.T) {
-		// TODO: Once it exists, use proper database snapshot function to freeze database, set backup id, and create volume snapshots
-
```

Comment:
You removed the TODO, but did not actually implement it. Snapshotting a running SM is not likely to break the test, but it is safer to do it right. Also, it would be nice to reduce how many places in the code hard code the location and format of `backup.txt`.

Reply:
The tests is exercising negative cases for `nuosm` where the archive is in an unexpected state. The backup hooks will not leave the archive in a state that can exercise these cases, so I don't think the TODO is relevant here.

Reply:
Out of the 8 snapshots, 4 are ones that `SnapshotSm` can handle (correct and incorrect `backupid` and with and without a journal). With the other 4, you need to muck around with the SM after the `pre-backup` script but before taking the snapshot: for 2 of them, you delete `backup.txt`, one with the archive directory moved, and one with a duplicate journal.